lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -307,4 +308,29 @@ def cmd_log(*args)
     end
   end
 
+  #
+  # Time how long in seconds a command takes to execute
+  #
+  def cmd_time(*args)
+    start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+    command = args.join(' ')
```

Comment:
I don't believe we need to use `Shellwords.join` here, but I could be convinced otherwise 😄

Reply:
This looks like the right call to me.

Given the above thought, if was there a simple way to just pass the args as an array instead of joining and re-parsing I would be all for it, however looking at the existing code path it would not save much in the the way of work and add more place we have maintain.